lib/3scale/backend/listener.rb

```diff
@@ -420,6 +422,10 @@ def do_api_method(method_name)
         transactions = params[:transactions]
         check_transactions_validity(transactions)
 
+        transactions.values.each do |tr|
```

Comment:
`tr` for transaction? I think we've been using `txn` or `tx` elsewhere, in case you'd want to check

Reply:
I don't see any `txn` or `tx`.

Reply:
Ok, we have been using just `t`.